### ISSUE TITLE
[Bug] Alchemy Not Giving XP 2.1.5

### ISSUE THREAD

**Issue by ImNotShady** [mcMMO-Dev/mcMMO#3758]
I just updated from 2.1.1 to 2.1.5 after hearing there was a fix for the alchemy bug in 2.1.4.
I wiped all configs and allowed McMMO to generate fresh files.
This is all fresh default CFGs.
All player data has been temporarily removed to try to further isolate this issue with no improvement. 
After testing, making sure proper permissions were given, all other skills do still function properly. Alchemy still fails to grant XP, I checked the configs to see if there was an error in the Experience.yml or any other issue in the other files but everything still seems to be in order.
I cannot seem to find what would be causing the issue on my end if it is indeed fixed.

**Comment by nossr50** [comment]
The only thing I think would be causing this is conflict with another plugin
Do you use WG? If you do make sure its on version 7

**Comment by ImNotShady** [comment]
Using WG Beta 3 running on Paper 512.
I tried isolating a plugin one by one, leaving only worldguard and worldedit to be the only other plugins installed, and still getting the same issue. This is still on 2.1.5 with fresh CFGs.
In another issue you mentioned CFGs being changed, I regenerated fresh CFGs after installing latest 2.1.6 today. And it seems like alchemy does work in 2.1.6 just fine. Strange.
